docs/building-spark.md

```diff
@@ -163,11 +164,9 @@ the `spark-parent` module).
 
 Thus, the full flow for running continuous-compilation of the `core` submodule may look more like:
 
-```
- $ mvn install
- $ cd core
- $ mvn scala:cc
-```
+    $ mvn install
```

Comment:
What's the change here -- I assume it's still code-formatted, but it's indented now?